uk.co.compucorp.civicrm.hremails/email-templates/src/pages/sickness-request.civicrm.html

```diff
@@ -0,0 +1,64 @@
+---
+layout: leave-absences
+subject: CiviHR Sickness Request
+request-type: Sickness
+---
+
+<div class="request request-sickness">
+  <row class="collapse">
+    <columns>
+      {{> callout-header title="Leave Request Type"}}
+      <callout class="request-data">
+        {{> request-data key="Status" value="{$leaveStatus}"}}
+        {{> request-data key="Staff Member" value="{contact.display_name}"}}
+
+        {if $leaveRequest->from_date eq $leaveRequest->to_date}
+          {{> request-data key="Date" value="{$fromDate|truncate:10:''|crmDate} {$fromDateType}"}}
+        {else}
+          {{> request-data key="From Date" value="{$fromDate|truncate:10:''|crmDate} {$fromDateType}"}}
```

Comment:
Well I saw this code a couple of places in L&A. Basically crmDate function formats a date in this form `2017-04-21 22:17:54` , the truncate removes the time part and leaves only the date. The from_date column is a DATE type.